code/__DEFINES/reagents.dm

```diff
@@ -0,0 +1,334 @@
+//Reagents
+
+#define VAPORSALT			"vaporsalt"
```

Comment:
> i copy pasted it
don't start shit

m8, it's my job to start shit.

> P.S. tabs for life

Wrong. Tabs for indentation, spaces for alignment.